### PR TITLE
Totals

### LINKED ISSUE #6: Mention `--html` in README.md

### PR DESCRIPTION
**Pull request by piuccio**
Hi

This other commit introduces totals. In my workflow I generate several CSS files and I'd like to check which module gives the better result across all files.
Just run `./bin/bench --total`

This PR is based on #27, if you merge it as it is you'll get that one for free :smile: 

It also mentions `--html` in the README so it fixes #6

### CODE REVIEW

**Comment by jakubpawlowicz** [comment]
@piuccio could you please rebase it on top of master?

**Comment by piuccio** [comment]
:smile: done

**Comment by jakubpawlowicz** [comment]
:+1: will review

File: bin/bench [hunk 1]
```diff
@@ -10,11 +10,11 @@ process.stdout.write = function() {};
 var only = process.argv.indexOf('--only') > -1 ?
   new RegExp('.*(' + process.argv[process.argv.indexOf('--only') + 1].replace(/,/g, '|') + ').*') :
   /.+/;
-var asHTML = process.argv.indexOf('--html') > -1;
```
Comment (XhmikosR) [inline bin/bench]: IMO having it as vars here seems better for maintenance.

Comment (piuccio) [inline bin/bench]: Fair enough. Indeed in the last pull request I converted them back to vars.

**Comment by jakubpawlowicz** [comment]
Looks good to me beside some formatting - will fix manually.

**Pull request merged**